core/cloudflow-akka/src/main/scala/cloudflow/akkastream/AkkaStreamletContextImpl.scala

```diff
@@ -174,6 +240,49 @@ final class AkkaStreamletContextImpl(
       .mapMaterializedValue(_ ⇒ NotUsed)
   }
 
+  def shardedPlainSource[T, E](
+      inlet: CodecInlet[T],
+      typeKey: EntityTypeKey[E],
+      entityIdExtractor: E => String,
+      resetPosition: ResetPosition = Latest
+  ): (Source[T, NotUsed], Future[KafkaClusterSharding.KafkaShardingNoEnvelopeExtractor[E]]) = {
```

Comment:
Would be great if this would follow the same approach as the shardedSourceWithCommittableContext, which you are probably planning.